modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -249,6 +249,12 @@ template<typename... Ts> inline GRunArgsP gout(Ts&... args)
     return GRunArgsP{ GRunArgP(detail::wrap_host_helper<Ts>::wrap_out(args))... };
 }
 
+// NB: Used by python wrapper
+struct GTypeInfo;
+using GTypesInfo = std::vector<GTypeInfo>;
+
+using ExtractArgsCallback = std::function<cv::GRunArgs(const cv::GTypesInfo& info)>;
```

Comment:
Is it relevant to `garg.hpp` at all?

Reply:
I don't think so, maybe we need a special place to move python stuff to there